exercises/binary-search/src/test/scala/BinarySearchTest.scala

```diff
@@ -0,0 +1,74 @@
+import org.scalatest.{FunSuite, Matchers}
+
+class BinarySearchTest extends FunSuite with Matchers {
+  test("finds a value in an array with one element") {
+    pending
+    val elements = Array[Int](6)
+    val result = BinarySearch.search(elements, 6)
+    assert(result.contains(0))
```

Comment:
IntelliJ was complaining about Some(0). I forget the actual complaint.
I think I like Some(0) and None better. So, since you agree I'll ignore IntelliJ and go back to Some/None.